app/logic/Mail/EWS/EWSAccount.ts

```diff
@@ -91,7 +92,10 @@ export class EWSAccount extends MailAccount {
   }
 
   async send(email: EMail): Promise<void> {
-    let request = new EWSCreateItemRequest({MessageDisposition: "SendAndSaveCopy"});
+    if (email.folder?.account != this) {
+      email.folder = this.getSpecialFolder(SpecialFolder.Sent);
+    }
```

Comment:
```suggestion
    assert(email.folder?.id, "Need folder to save the sent email in");
```

Reply:
(This change makes the import of `SpecialFolder` unnecessary.)